stream-chat-android-core/src/main/java/io/getstream/chat/android/models/GuestUser.kt

```diff
@@ -16,4 +16,7 @@
 
 package io.getstream.chat.android.models
 
+import androidx.compose.runtime.Stable
+
+@Stable
```

Comment:
I'm wondering if this can be `@Immutable`?

Reply:
If this model doesn't need to be observed or tracked as a state by Compose runtime, we can mark this with an `Immutable` annotation.